tests/foreman/cli/test_logging.py

```diff
@@ -0,0 +1,278 @@
+"""CLI tests for logging.
+
+:Requirement: Logging
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: logging
+
+:TestType: Functional
+
+:CaseImportance: Medium
+
+:Upstream: No
+"""
+import re
+import logging
+from fauxfactory import gen_string
+from robottelo import ssh
+from robottelo.ssh import upload_file
+from robottelo import manifests
+from robottelo.test import CLITestCase
+from robottelo.cli.factory import (
+    make_org,
+    make_product_wait,
+    make_repository,
+)
+from robottelo.cli.subscription import Subscription
+log = logging.getLogger(__name__)
```

Comment:
Am I blind, or you don't actually use this?
`self.logger` is from https://github.com/SatelliteQE/robottelo/blob/master/robottelo/test.py#L178

Reply:
OK, removed. Thank you.